105/105-hw/UIKitHomework/UIKitHomework/BrowseCatalogViewController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  BrowseCatalogViewController.swift
+//  UIKitHomework
+//
+//  Created by Artyom Mitrofanov on 08.10.2022.
+//
+
+import UIKit
+
+var catalogNumber: Int = 1
+
+class BrowseCatalogViewController: UIViewController {
+
+    @IBOutlet var closeButton: UIButton!
+    @IBOutlet var catalogPageLabel: UILabel!
+    @IBOutlet var browseCatalogButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        catalogPageLabel.text = "Catalog, Page \(catalogNumber)"
+        catalogPageLabel.font = UIFont.systemFont(ofSize: 25)
+        setupConstrains()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        self.navigationController?.isNavigationBarHidden = true
+    }
+
+    
+    private func setupConstrains() {
+        browseCatalogButton.translatesAutoresizingMaskIntoConstraints = false
+        catalogPageLabel.translatesAutoresizingMaskIntoConstraints = false
+        closeButton.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate([
+            browseCatalogButton.centerXAnchor.constraint(equalTo:
+                                                            view.centerXAnchor,
+                                                         constant: 0),
+            browseCatalogButton.centerYAnchor.constraint(equalTo:
+                                                            view.centerYAnchor,
+                                                         constant: 0),
+            browseCatalogButton.trailingAnchor.constraint(equalTo:
+                                                            view.trailingAnchor,
+                                                          constant: -20),
+            browseCatalogButton.leadingAnchor.constraint(equalTo:
+                                                            view.leadingAnchor,
+                                                         constant: 20),
+            browseCatalogButton.heightAnchor.constraint(equalToConstant: 50)
+        ])
```

Comment:
С такими переносами код плохо читается. Давай по-другому:
```suggestion
        NSLayoutConstraint.activate([
            browseCatalogButton.centerXAnchor.constraint(
                equalTo: view.centerXAnchor,
                constant: 0
            ),
            browseCatalogButton.centerYAnchor.constraint(
                equalTo: view.centerYAnchor,
                constant: 0
            ),
            browseCatalogButton.trailingAnchor.constraint(
                equalTo: view.trailingAnchor,
                constant: -20
            ),
            browseCatalogButton.leadingAnchor.constraint(
                equalTo: view.leadingAnchor,
                constant: 20
            ),
            browseCatalogButton.heightAnchor.constraint(equalToConstant: 50)
        ])
```